progman.c

```diff
@@ -5,7 +5,7 @@
 #include "serial.h"
 #include "motion_control.h"
 
-#define PROGRAM_MAX_SIZE (1024 * 3)
+#define PROGRAM_MAX_SIZE (2560)
```

Comment:
Good question.  It was originally 3k, I reduced it to 2.5k. It was arbitrarily chosen to hold a large amount of gcode to handle a "worst case" key + some additional space. I'll remove it from this PR though since it doesn't really belong.